android-lib/src/androidTest/java/com/kinvey/androidTest/store/DataStoreTest.java

```diff
@@ -1428,20 +1447,18 @@ public void testPullNotCorrectItem() throws InterruptedException {
         CustomKinveyPullCallback<Person> pullCallback = testManager.pullCustom(store, null);
         assertTrue(pullCallback.getResult().getListOfExceptions().size() == 1);
         assertTrue(pullCallback.getResult().getResult().size() == 4);
-        testManager.cleanBackendDataStore(store);
+        testManager.cleanBackend(store, StoreType.SYNC);
     }
 
     @Test
     public void testPagedPullNotCorrectItem() throws InterruptedException {
         TestManager<Person> testManager = new TestManager<>();
         testManager.login(TestManager.USERNAME, TestManager.PASSWORD, client);
         DataStore<Person> store = DataStore.collection(Person.COLLECTION_WITH_EXCEPTION, Person.class, StoreType.SYNC, client);
-        store.setAutoPagination(true);
-        store.setAutoPaginationPageSize(2);
-        CustomKinveyPullCallback<Person> pullCallback = testManager.pullCustom(store, null);
+        CustomKinveyPullCallback<Person> pullCallback = testManager.pullCustom(store, null, 2);
         assertTrue(pullCallback.getResult().getListOfExceptions().size() == 1);
         assertTrue(pullCallback.getResult().getResult().size() == 4);
-        testManager.cleanBackendDataStore(store);
+        testManager.cleanBackend(store, StoreType.SYNC);
```

Comment:
I've removed this line because the test uses a special collection which contains not correct item, the collection is updated never. Also, items in the collection were created using master key so user from the test can't delete them.